docs/root/start/sandboxes/mysql.rst

```diff
@@ -91,17 +91,18 @@ Terminal 1
 
 .. code-block:: console
 
-  $ curl -s http://localhost:8001/stats?filter=egress_mysql
+  $ curl -s "http://localhost:8001/stats?filter=egress_mysql"
   mysql.egress_mysql.auth_switch_request: 0
   mysql.egress_mysql.decoder_errors: 0
   mysql.egress_mysql.login_attempts: 1
   mysql.egress_mysql.login_failures: 0
-  mysql.egress_mysql.protocol_errors: 0
-  mysql.egress_mysql.queries_parse_error: 0
+  mysql.egress_mysql.protocol_errors: 21
+  mysql.egress_mysql.queries_parse_error: 2
```

Comment:
If you felt surprise for `mysql.egress_mysql.protocol_errors: 21`, ref this issue #14975.

Reply:
Yeah, IMHO the filter should not be stateful and check expected seq. It should just do parsing.